website/docs/r/federated_settings_identity_provider.html.markdown

```diff
@@ -31,15 +33,21 @@ resource "mongodbatlas_federated_settings_identity_provider" "identity_provider"
 
 * `federation_settings_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
 * `name` - (Required) Human-readable label that identifies the identity provider.
-* `associated_domains` - (Required) List that contains the domains associated with the identity provider.
-* `sso_debug_enabled` - (Required) Flag that indicates whether the identity provider has SSO debug enabled.
-* `status`- (Required) String enum that indicates whether the identity provider is active or not. Accepted values are ACTIVE or INACTIVE.
-* `issuer_uri` - (Required) Unique string that identifies the issuer of the SAML
-* `sso_url` - (Required) Unique string that identifies the intended audience of the SAML assertion.
-* `request_binding` - (Required) SAML Authentication Request Protocol HTTP method binding (POST or REDIRECT) that Federated Authentication uses to send the authentication request. Atlas supports the following binding values:
+* `associated_domains` - List that contains the domains associated with the identity provider.
+* `sso_debug_enabled` - Flag that indicates whether the identity provider has SSO debug enabled.
+* `status`- String enum that indicates whether the identity provider is active or not. Accepted values are ACTIVE or INACTIVE.
+* `issuer_uri` - (Required) Unique string that identifies the issuer of the IdP
+* `sso_url` - Unique string that identifies the intended audience of the SAML assertion.
+* `request_binding` - SAML Authentication Request Protocol HTTP method binding (POST or REDIRECT) that Federated Authentication uses to send the authentication request. Atlas supports the following binding values:
     - HTTP POST
     - HTTP REDIRECT
-* `response_signature_algorithm` - (Required) Signature algorithm that Federated Authentication uses to encrypt the identity provider signature.  Valid values include SHA-1 and SHA-256.
+* `response_signature_algorithm` - Signature algorithm that Federated Authentication uses to encrypt the identity provider signature.  Valid values include SHA-1 and SHA-256.
+* `protocol` - The protocol of the identity provider. Either SAML or OIDC.
```

Comment:
maybe it would be good in each attribute to say if it's mandatory depending on the protocol.    like... this is mandatory for SAML protocol.

Reply:
Maybe required would not be the right word, since it's not actually required, I'm not sure about it